client/src/pages/ProfileSkeleton/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,62 @@
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import ProfileSelection from '../../../components/ProfileSelection/ProfileSelection';
+import Avatar from '@mui/material/Avatar';
+import { useAuth } from '../../../context/useAuthContext';
+import Button from '@material-ui/core/Button';
+import { CircularProgress } from '@material-ui/core';
+import { useForm } from 'react-hook-form';
+import uploadPhoto from '../../../helpers/APICalls/uploadPhoto';
+
+export default function ProfilePhoto(): JSX.Element {
+  const { register, handleSubmit } = useForm();
+  const classes = useStyles();
+  const { userProfile } = useAuth();
+  console.log(!userProfile ? 0 : userProfile.photoUrl.substring(72));
+  return (
+    <Grid container className={classes.root}>
+      <Grid item xs={6} sm={6} md={5} elevation={0} component={Paper} square className={classes.back}>
+        <Box className={classes.authWrapper}>
+          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
+            <ProfileSelection name={'Edit profile'} link={'/profileEditionMock'} />
+            <ProfileSelection name={'Profile photo'} link={'/profilephoto'} />
+            <ProfileSelection name={'Availability'} link={'/profileEditionMock'} />
+            <ProfileSelection name={'Payment'} link={'/profileEditionMock'} />
+            <ProfileSelection name={'Security'} link={'/profileEditionMock'} />
+            <ProfileSelection name={'Settings'} link={'/profileEditionMock'} />
+          </Box>
+        </Box>
+      </Grid>
+      <Grid item xs={6} sm={3} md={6} elevation={6} component={Paper} square>
+        <Box className={classes.authWrapper}>
+          <Box width="100%" maxWidth={350} p={3} alignSelf="center">
+            <Typography className={classes.welcome} component="h1" variant="h5">
+              Profile Photo
+            </Typography>
+            <Avatar
+              alt="Profile Image"
+              src={
+                userProfile && userProfile.photoUrl
+                  ? `../../../Images/${userProfile.photoUrl.substring(72)}`
```

Comment:
@rajivtitus here it is. the avatar display
![39](https://user-images.githubusercontent.com/86894074/139739354-40e6034b-78dd-4018-8e1d-c1d9e7c8c978.png)

Reply:
Any one reviewing here please give me hint.
My logic is if the UserProfile and its photourl both exist then render the photo from the local repo. Otherwise, it renders a default file in the same location.
But now it looks like the location is considered as a URL......

Reply:
@JAFAJ0 
Why aren't we using the URL which we will receive after uploading an image to Amazon S3?
Also, if you want to use a local image as the source for an image element/component, you need to import it within this component.

Reply:
@rajivtitus Fix is by making S3 public to read